MediaGalleryUi/view/adminhtml/ui_component/media_gallery_listing.xml

```diff
@@ -34,7 +34,7 @@
             </settings>
         </dataProvider>
     </dataSource>
-    <listingToolbar name="listing_top">
+    <listingToolbar name="listing_top" template="Magento_AdobeStockImageAdminUi/grid/toolbar">
```

Comment:
media gallery should not depend on Magento_AdobeStockImageAdminUi, we have to either copy the template or move it to Magento_Ui module to use in both modules.